800.renames-and-merges/e.yaml

```diff
@@ -78,6 +78,7 @@
 - { setname: emacs,                    name: [emacs-devel-nox], addflavor: nox11, weak_devel: true, nolegacy: true }
 - { setname: emacs,                    name: emacs-git-gtk2, addflavor: gtk2, ignore: true }
 - { setname: emacs,                    namepat: "emacs[0-9.-]+(-non-dfsg)?" }
+- { setname: emacs,                    name: emacs-console }
```

Comment:
addflavor

Reply:
There isn't a package called `emacs` in Chimera Linux, and emacs is called `emacs-console` in that repository. `addflavor` isn't applicable here.

Reply:
Ruleset does not need to be aware of whether there are other flavors of the same project in repository, and also this can change in the future. So if a name looks flavored, it needs `addflavor`.